File: src/main/java/org/game/Hard.java
package org.game;

import javax.swing.*;

/**
 * 32 x 32, 256 mines
 */

public final class Hard implements GameType {
    private Hard() { }

    /**
     * 32 x 32, 256 mines
     * @param name name of player
     */

    public static final void start(final String name, final String password) {
        JOptionPane.showMessageDialog(
                null,
                "Good luck, " + name + "!",
                "Greetings",
                JOptionPane.INFORMATION_MESSAGE
        );

        new Game(32, 32, 256, name, password, new Hard()).start();
    }
}

File: src/main/kotlin/org/game/Hard.kt
package org.game

import javax.swing.JOptionPane

/**
 * 32 x 32, 256 mines
 */


internal class Hard : GameType {
    companion object {

        /**
         * 32 x 32, 256 mines
         * @param name name of player
         */


        fun start(name: String, password: String) {
            JOptionPane.showMessageDialog(
                null,
                "Good luck, $name!",
                "Greetings",
                JOptionPane.INFORMATION_MESSAGE
            )

            Game(32, 32, 256, name, password, Hard()).start()
        }
    }
}